src/index/repositories.props

```diff
@@ -13,6 +13,15 @@
         "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat" &amp; build -SkipTests
       </PrepareCommand>
     </Repository>
+    <Repository Include="corefxlab">
+      <Url>https://github.com/dotnet/corefxlab</Url>
+      <Projects>
+        src/*.csproj;
+      </Projects>
+      <PrepareCommand>
+        "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat" &amp; build
```

Comment:
If this will run any tests please add the correct parameters to disable them.

Reply:
I don't think there's currently a way to disable tests in corefxlab. Is that a blocker?

Reply:
Not really, it just causes things to be slower.